modin/pandas/test/test_series.py

```diff
@@ -3389,54 +3389,54 @@ def test_update(data, other_data):
     df_equals(modin_series, pandas_series)
 
 
-@pytest.mark.parametrize("normalize, bins, dropna", [(True, 3, False)])
-def test_value_counts(normalize, bins, dropna):
-    # We sort indices for Modin and pandas result because of issue #1650
-    modin_series, pandas_series = create_test_series(test_data_values[0])
-
-    modin_result = sort_index_for_equal_values(
-        modin_series.value_counts(normalize=normalize, ascending=False), False
-    )
-    pandas_result = sort_index_for_equal_values(
-        pandas_series.value_counts(normalize=normalize, ascending=False), False
-    )
-    df_equals(modin_result, pandas_result)
-
-    modin_result = sort_index_for_equal_values(
-        modin_series.value_counts(bins=bins, ascending=False), False
-    )
-    pandas_result = sort_index_for_equal_values(
-        pandas_series.value_counts(bins=bins, ascending=False), False
-    )
-    df_equals(modin_result, pandas_result)
+@pytest.mark.parametrize("sort", bool_arg_values, ids=bool_arg_keys)
+@pytest.mark.parametrize("normalize", bool_arg_values, ids=bool_arg_keys)
+@pytest.mark.parametrize("bins", [3, None])
+@pytest.mark.parametrize("dropna", bool_arg_values, ids=bool_arg_keys)
+@pytest.mark.parametrize("ascending", bool_arg_values, ids=bool_arg_keys)
+def test_value_counts(sort, normalize, bins, dropna, ascending):
+    def sort_sensitive_comparator(df1, df2):
+        return (
+            df_equals_with_non_stable_indices(df1, df2)
+            if sort
+            else df_equals(df1.sort_index(), df2.sort_index())
+        )
 
-    modin_result = sort_index_for_equal_values(
-        modin_series.value_counts(dropna=dropna, ascending=True), True
-    )
-    pandas_result = sort_index_for_equal_values(
-        pandas_series.value_counts(dropna=dropna, ascending=True), True
+    # We sort indices for Modin and pandas result because of issue #1650
```

Comment:
I think this comment should go in `sort_sensitive_comparator`, not stay underneath it

Reply:
right, moved this comment to the function